src/features/form/validations.js

```diff
@@ -9,9 +11,16 @@ const abnormalMaxValue = max => value =>
 
 const abnormalMinValue = min => value => (value && value < min ? `Abnormal value` : undefined);
 
+const dateToInt = dateStr => new Date(dateStr).getTime();
+
+const maxDateValue = maxDate => value =>
+  value && (dateToInt(value)) > dateToInt(maxDate) ? `Date cannot be in the future` : undefined;
```

Comment:
It looks like this is a generic validator that supports testing against any specific maxDate?  This is good, but the validation message is inaccurate in this case (always says "cannot be in the future")